packages/ui/package.json

```diff
@@ -21,37 +21,44 @@
 		"prepublishOnly": "pnpm run package",
 		"prepare": "svelte-kit sync",
 		"storybook": "storybook dev --no-open -p 6006",
-		"storybook:build": "storybook build"
+		"storybook:build": "storybook build",
+		"test": "vitest run --mode development",
+		"test:watch": "vitest --watch --mode development"
 	},
 	"devDependencies": {
 		"@csstools/postcss-bundler": "^1.0.15",
-		"@storybook/addon-docs": "^8.2.7",
-		"@storybook/addon-essentials": "^8.2.7",
-		"@storybook/addon-links": "^8.2.7",
-		"@storybook/blocks": "^8.2.7",
-		"@storybook/svelte": "^8.2.7",
-		"@storybook/sveltekit": "^8.2.7",
+		"@storybook/addon-docs": "^8.3.0-beta.0",
+		"@storybook/addon-essentials": "^8.3.0-beta.0",
+		"@storybook/addon-links": "^8.3.0-beta.0",
+		"@storybook/blocks": "^8.3.0-beta.0",
+		"@storybook/experimental-addon-vitest": "^8.3.0-beta.0",
+		"@storybook/svelte": "^8.3.0-beta.0",
+		"@storybook/sveltekit": "^8.3.0-beta.0",
 		"@sveltejs/adapter-static": "catalog:svelte",
 		"@sveltejs/kit": "catalog:svelte",
 		"@sveltejs/package": "^2.3.2",
 		"@sveltejs/vite-plugin-svelte": "catalog:svelte",
 		"@terrazzo/cli": "^0.0.11",
 		"@terrazzo/plugin-css": "^0.0.9",
 		"@types/postcss-pxtorem": "^6.0.3",
+		"@vitest/browser": "^2.0.5",
 		"autoprefixer": "^10.4.19",
 		"cpy-cli": "^5.0.0",
 		"date-fns": "^2.30.0",
+		"playwright": "^1.46.1",
 		"postcss": "^8.4.38",
 		"postcss-cli": "^11.0.0",
 		"postcss-minify": "^1.1.0",
 		"postcss-nesting": "^12.1.5",
 		"postcss-pxtorem": "^6.1.0",
 		"rimraf": "^6.0.1",
-		"storybook": "^8.2.7",
+		"storybook": "^8.3.0-beta.0",
 		"storybook-dark-mode": "^4.0.2",
 		"svelte": "catalog:svelte",
 		"svelte-check": "catalog:svelte",
-		"vite": "catalog:"
+		"util": "^0.12.5",
```

Comment:
`util` here is a temporary workaround for https://github.com/storybookjs/storybook/pull/29008. Can be removed again when that fix has been released.